website/docs/cloud-docs/vcs/azure-devops-services.mdx

```diff
@@ -24,18 +24,19 @@ Configuring a new VCS provider requires permission to [manage VCS settings](/clo
 Before you begin, enable `Third-party application access via OAuth` in Azure DevOps Services settings.
 
 1. Log in to [Azure DevOps Services](https://dev.azure.com/).
-1. Click **Organization settings** in the bottom of the sidebar.
-1. Click **Policies** in the sidebar under **Security**.
+1. Click **Organization settings**.
```

Comment:
Just want to flag that we have moved beyond TFC here :) this is documenting things in the Azure DevOps Services product I believe. I am going to leave, as I think the changes are minimal enough that they'll be clear. but we should watch out for this in other VCS pages!